src/functions.js

```diff
@@ -1,14 +1,100 @@
 
 const path = require('path');
 const fs = require('fs');
+const tools = require('simple-svg-tools');
+const directoryPath = path.join(__dirname, 'svgs');
+const SVGO = require('svgo');
+const exportPath = path.join(__dirname, 'optimizedSVGs');
 
-function getAllFileName(){ 
-    const directoryPath = path.join(__dirname, 'svgs');
-    fs.readdir(directoryPath, function (err, files) {
-        if (err) {
-            return console.log('Unable to scan directory: ' + err);
-        }
-    return files;
+const svgo = new SVGO({
+    plugins: [{
+       cleanupAttrs: true,
+   }, {
+       removeDoctype: true,
+   },{
+       removeXMLProcInst: true,
+   },{
+       removeComments: true,
+   },{
+       removeMetadata: true,
+   },{
+       removeTitle: true,
+   },{
+       removeDesc: true,
+   },{
+       removeUselessDefs: true,
+   },{
+       removeEditorsNSData: true,
+   },{
+       removeEmptyAttrs: true,
+   },{
+       removeHiddenElems: true,
+   },{
+       removeEmptyText: true,
+   },{
+       removeEmptyContainers: true,
+   },{
+       removeViewBox: false,
+   },{
+       cleanupEnableBackground: true,
+   },{
+       convertStyleToAttrs: true,
+   },{
+       convertColors: true,
+   },{
+       convertPathData: true,
+   },{
+       convertTransform: true,
+   },{
+       removeUnknownsAndDefaults: true,
+   },{
+        removeNonInheritableGroupAttrs: true,
+   },{
+       removeUselessStrokeAndFill: true,
+   },{
+       removeUnusedNS: true,
+   },{
+        cleanupIDs: true,
+   },{
+       cleanupNumericValues: true,
+   },{
+       moveElemsAttrsToGroup: true,
+   },{
+       moveGroupAttrsToElems: true,
+   },{
+       collapseGroups: true,
+   },{
+       removeRasterImages: false,
+   },{
+       mergePaths: true,
+   },{
+       convertShapeToPath: true,
+   },{
+       sortAttrs: true,
+   },{
+       removeDimensions: true,
+   },{
+       removeAttrs: {attrs: '(stroke|fill)'},
+   }]
+});
```

Comment:
export this plugin json to a svgo config file please, then import it into this main js file